### ISSUE TITLE
[Legitimate Site Blocked] rexai.xyz

### ISSUE THREAD

**Issue by ripreus** [MetaMask/eth-phishing-detect#28947]
Hello, we are a team of 20 - 25-year-olds who stepped into the crypto industry and foresee the future of blockchain software. That's why we know that as we make progress on this path, we will inevitably encounter mistakes and we will correct them. However, the phishing violation in question does not reflect the truth at all and we think that the system perceives it as such because it is an error caused by the web3 module we are trying to integrate into our website. Necessary precautions and corrections have been made by our team, you can check with peace of mind. Please remove this domain name from the blacklist as soon as possible for the sake of our future project. https://rexai.xyz/

**Comment by AlexHerman1** [comment]
Hi there,

Is there any social media presence for your team or the product itself? Are the team members on the website?

**Comment by ripreus** [comment]
Hello Alex, as we mentioned, we are currently only in the construction phase. We'll be on social media soon, but we're trying to make sure everything is smooth, legal, and accurate for that to happen. So, we are just at the beginning and we do not use personal social media accounts, but our website's social media accounts are available as a draft. In this process, we have to maintain the reliability of our website's domain name for the continuation of our project. We have already fixed this minor issue and will be waiting for a positive response from you as well.

**Comment by AlexHerman1** [comment]
Hi,

We do not unblock sites that are reported for phishing previously and have nothing hosted. if you get something stood up here you can open a new issue in the future. Im not sure what minor issue you're referring to.
